src/main/resources/application.yaml

```diff
@@ -38,9 +38,18 @@ quarkus:
         url: ldap://override-when-needed
       identity-mapping:
         search-base-dn: ou=users,o=YourCompany,c=ES
-# Quartz configuration
+  # Flyway to create PIM schema
+  flyway:
+    connect-retries: 10
+    table: flyway_pim_history
+    migrate-at-start: true
+    baseline-on-migrate: true
+    baseline-version: 1.0
+    baseline-description: PimDB
+    sql-migration-prefix: h2
+  # Quartz configuration
   quartz:
-    store-type: ram
+    store-type: jdbc-cmt
```

Comment:
Shouldn't it be jdbc-tx or ram? (according to README)

Reply:
It should be `jdbc-cmt` - In the README, it is stating that the quartz is using a in-memory database managed by the container, which is `jdbc-cmt`. I will update the README file to clarify this. Thanks!